apps/ledger-live-mobile/src/dynamicContent/ContentCardsCategory/Layout.tsx

```diff
@@ -10,10 +10,18 @@ import {
 import { Flex } from "@ledgerhq/native-ui";
 import { ContentCardMetadata } from "~/contentCards/cards/types";
 import { contentCardItem } from "~/contentCards/cards/utils";
-import { compareCards, mapAsHorizontalContentCard } from "~/dynamicContent/utils";
+import {
+  compareCards,
+  mapAsHorizontalContentCard,
+  mapAsSmallSquareContentCard,
+  mapAsMediumSquareContentCard,
+  mapAsBigSquareContentCard,
+} from "~/dynamicContent/utils";
 import Carousel from "../../contentCards/layouts/carousel";
 import useDynamicContent from "../useDynamicContent";
 import { ContentCardsType } from "../types";
+import Grid from "~/contentCards/layouts/grid";
+import VerticalCard from "~/contentCards/cards/vertical";
 
 // TODO : Better type to remove any (maybe use AnyContentCard)
 /* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Is this comment still relevant? (As I see that we use AnyContentCard)

Reply:
Still relevant. We still have an `any` that we need to find a way to remove